admin/app/components/complementary-certifications/attach-badges/badges/index.js

```diff
@@ -21,6 +22,10 @@ export default class Badges extends Component {
     return 'Seul un profil cible comportant au moins un résultat thématique certifiant peut être rattaché à une certification complémentaire. Le profil cible que vous avez sélectionné ne comporte pas de résultat thématique certifiant. Veuillez le modifier puis rafraîchir cette page ou bien sélectionner un autre profil cible.';
   }
 
+  get getCertiableBadges() {
```

Comment:
Le "getXXX" suppose que c'est une methode, le getter simule un attribut, je propose de mettre directement `certifiableBadges`
```suggestion
  get certiableBadges() {
```